integration-tests/protocolv2_test.go

```diff
@@ -0,0 +1,171 @@
+package integration_tests
+
+import (
+	"context"
+	"fmt"
+	"github.com/datastax/go-cassandra-native-protocol/client"
+	"github.com/datastax/go-cassandra-native-protocol/datatype"
+	"github.com/datastax/go-cassandra-native-protocol/frame"
+	"github.com/datastax/go-cassandra-native-protocol/message"
+	"github.com/datastax/go-cassandra-native-protocol/primitive"
+	"github.com/datastax/zdm-proxy/integration-tests/setup"
+	"github.com/stretchr/testify/require"
+	"net"
+	"testing"
+)
+
+func TestProtocolV2Connect(t *testing.T) {
```

Comment:
Remember that we talked about having these tests next to the protocol version negotiation ones

Reply:
Offered some other suggestions for these tests  through slack